packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1728,6 +1738,16 @@ export default class NewVm extends BaseComponent {
                 value={cpuCap}
               />
             </Item>
+            <Item label={_('cpusMax')}>
+              <DebounceInput
+                className='form-control'
+                debounceTimeout={3e3}
+                min={CPUs}
+                onChange={this._linkState('cpusMax')}
+                type='number'
+                value={this._getCpusMax()}
```

Comment:
Here's a scenario:
- I set `CPUs` to 8
- I set `cpusMax` to 4
- `cpusMax` is automatically set back to 8
- I set `CPUs` to 2
- `cpusMax` is automatically set to 4

The last step feels weird, is that what we want?

Reply:
Right :+1: , so, i think we should display the current `CPUs` max value and use the computed value for computation.